ios/FluentUI/Vnext/Button/MSFButton.swift

```diff
@@ -49,10 +49,10 @@ import UIKit
 
     private func setupLargeContentViewer() {
         let largeContentViewerInteraction = UILargeContentViewerInteraction()
-        view.addInteraction(largeContentViewerInteraction)
+        self.addInteraction(largeContentViewerInteraction)
         largeContentViewerInteraction.gestureRecognizerForExclusionRelationship.delegate = self
-        view.scalesLargeContentImage = true
-        view.showsLargeContentViewer = state.style.isFloatingStyle
+        self.scalesLargeContentImage = true
+        self.showsLargeContentViewer = state.style.isFloatingStyle
```

Comment:
Do we need all the `self`s?

Reply:
That said, what's the convention? The rest of this file uses `self.` to access properties on the superclass, so I was mostly following the file's convention.

Reply:
Gotcha. I think I'm gonna keep it as is for now, just to stay consistent in the file with other places, but I agree that there's possibly some improvement that could be done.